.github/workflows/create-release.yml

```diff
@@ -12,6 +12,11 @@ on:
         description: 'RunId of liquibase/liquibase artifacts to attach'
         required: true
 
+env:
+  MAVEN_VERSION: '3.9.2'
```

Comment:
I think we don't need maven version + stCarolas down there as we are only getting dependencies with maven. we can use the one provided by image.